src/diffusers/models/resnet.py

```diff
@@ -97,7 +247,19 @@ def __init__(
         conv_2d_out_channels: Optional[int] = None,
     ):
         super().__init__()
-        self.pre_norm = pre_norm
+        if time_embedding_norm == "ada_group":
+            deprecate(
+                "ada_group",
+                "1.0.0",
+                "Passing `ada_group` as `time_embedding_norm` is deprecated, please create `ResnetBlockCondNorm2D` instead",
+            )
+        if time_embedding_norm == "spatial":
+            deprecate(
+                "spatial",
+                "1.0.0",
+                "Passing `spatial` as `time_embedding_norm` is deprecated, please create `ResnetBlockCondNorm2D` instead",
+            )
```

Comment:
```suggestion
            raise ValueError(
                "spatial",
                "1.0.0",
                "Passing `spatial` as `time_embedding_norm` is deprecated, please create `ResnetBlockCondNorm2D` instead",
            )
```

I think we can just raise an error here because these are internal classes. If we deprecate we can't just ignore the argument. In this case we would have to return a `ResnetBlockCondNorm2D` class